sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormRecognizerAsyncClient.java

```diff
@@ -31,11 +32,15 @@
 import java.util.function.Function;
 
 import static com.azure.ai.formrecognizer.Transforms.toReceipt;
+import static com.azure.ai.formrecognizer.Transforms.toRecognizedFormSupervised;
+import static com.azure.ai.formrecognizer.Transforms.toRecognizedLayout;
+import static com.azure.ai.formrecognizer.implementation.Utility.parseModelId;
 import static com.azure.core.util.FluxUtil.monoError;
 
 /**
  * This class provides an asynchronous client that contains all the operations that apply to Azure Form Recognizer.
- * Operations allowed by the client are, to extract receipt data fields from receipt documents.
+ * Operations allowed by the client are, to recognize receipt data from documents, extract layout information and
+ * analyze custom forms for predefined data.
```

Comment:
No need to add a coma before listing the allowed operations.

```suggestion
 * Operations allowed by the client are recognizing receipt data from documents, extracting layout information and
 * analyzing custom forms for predefined data.
```